bench/wtperf/config.c

```diff
@@ -610,11 +595,10 @@ config_opt_line(CONFIG *cfg, const char *optstr)
 	 * any parsed from the original config. We allocate len + 1 to allow for
 	 * a null byte to be added.
 	 */
-	if ((string_copy = calloc(len + 1, 1)) == NULL)
-		return (enomem(cfg));
+	string_copy = dcalloc(len + 1, 1);
 
-	strncpy(string_copy, optstr, len);
-	config_line = calloc(sizeof(CONFIG_QUEUE_ENTRY), 1);
+	string_copy = dstrdup(optstr);
+	config_line = dcalloc(sizeof(CONFIG_QUEUE_ENTRY), 1);
 	config_line->string = string_copy;
```

Comment:
Another example, I think this would be:
```
config_line = dcalloc(sizeof(CONFIG_QUEUE_ENTRY), 1);
config_line->string = dstrdup(optstr);
```
and `string_copy` isn't needed.